holidays/locale/en_US/LC_MESSAGES/AR.po

```diff
@@ -16,111 +16,113 @@ msgstr ""
 "Generated-By: pygettext.py 1.5\n"
 "X-Generator: Poedit 3.2.2\n"
 
-#: holidays/countries/argentina.py:71
+#: ./holidays/countries/argentina.py:69
```

Comment:
Same here, the leading ./ look unnecessary. Might be a good idea to revert all the .po files alltogether.